packages/aws-cdk/README.md

```diff
@@ -1145,6 +1145,36 @@ The pattern language is the same as the one used in the `cdk deploy` command.
 However, unlike `cdk deploy`, in the absence of this parameter, all stacks are 
 considered.
 
+If, instead of letting the CLI decide which resources to move, you want to 
+provide your own mapping of old to new locations, you can do so by passing a
+mapping file to the CLI via the `--mapping-file` flag. This file should 
+contain a JSON object with the following format: 
+
+```json
+{
+  "mappings": [
+    {
+      "source": "Foo.OldName",
+      "destination": "Bar.NewName",
+      "environment": {
+        "name": "test",
+        "account": "123456789012",
+        "region": "us-east-1"
+      }
```

Comment:
Oof! If we need to pass the environment for every resource, that is going to be an annoying file to work with if the list of resources grows, right?

How about something like this:

```json
{ 
  "environments": [
    {
      "account": "123456789012",
      "region": "us-east-1",
      "resources": {
        "Bar.NewName": "Foo.OldName"
      } 
    }
  ]
}
```

(Or reverse the mapping, old-to-new might be more logical to people's minds)